### PR TITLE
DAMIEN-458, /dept and /status accessibility: SR alert when form element change; more responsive UI

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira-secure.berkeley.edu/browse/DAMIEN-458

Again, I recommend `?w=1`

### CODE REVIEW

File: src/views/BaseView.vue [hunk 1]
```diff
@@ -66,43 +66,47 @@
...
+              <span class="sr-only">User profile for </span>{{ $currentUser.firstName }}
+            </v-btn>
+          </template>
+          <v-list>
+            <v-list-item id="menu-item-log-out" link @click="logOut">
+              <v-list-item-content>Log Out</v-list-item-content>
+            </v-list-item>
+          </v-list>
+        </v-menu>
+      </div>
```
Comment (johncrossman) [inline src/views/BaseView.vue:109]: I amended this PR with improvements in UI responsiveness. (Travis is green.)

File: src/components/evaluation/EvaluationTable.vue [hunk 2]
```diff
@@ -769,7 +771,9 @@ export default {
     toggleSelectAll() {
       if (this.allEvaluationsSelected || this.someEvaluationsSelected) {
         this.deselectAllEvaluations()
+        this.alertScreenReader('All evaluations unselected')
       } else {
+        this.alertScreenReader('All valuations selected')
```
Comment (lyttam) [inline src/components/evaluation/EvaluationTable.vue]: @johncrossman  just missing an 'e' on 'valuations'!

File: src/views/StatusBoard.vue [hunk 3]
```diff
@@ -243,18 +245,18 @@ export default {
...
       const index = this.$_.indexOf(this.selectedDepartmentIds, department.id)
-      if (index === -1) {
+      const isSelecting = index === -1
+      if (isSelecting) {
         this.selectedDepartmentIds.push(department.id)
       } else {
         this.selectedDepartmentIds.splice(index, 1)
       }
+      this.alertScreenReader(`${department.name} ${isSelecting ? '' : 'un'}selected`)
+      console.log('sss')
```
Comment (lyttam) [inline src/views/StatusBoard.vue]: @johncrossman Leftover log statement?

Comment (johncrossman) [reply src/views/StatusBoard.vue]: Good catch.  Amending

**Pull request merged**